config/tracks/solitude1964.ini

```diff
@@ -1,62 +1,578 @@
+[ABOUT]
+AUTHOR = leBluem, Sliderman
+VERSION = 1.01
+DATE_RELEASE = 2019 jan 4, 01.07.19, 03.12.19, 06.08.2021
+LIGHTS_COUNT = 
+NOTES = solitude1964 tweaked meshes, materials and lights, grass, water, rain, wind
+
+[BASIC]
+SUPPORTS_WIND = 1
+
+[WIND]
+;IGNORE =
+;DYNAMIC_FLAGS = AUTO
+
 [LIGHTING]
-LIT_MULT = 1
+LIT_MULT = 1.0
+CAR_LIGHTS_LIT_MULT = 1.0
+SPECULAR_MULT = 1.0
+
+[MODEL_REPLACEMENT_0]
+ACTIVE = 1
+DESCRIPTION = hide some objects like crowd
+
+[INCLUDE]
+INCLUDE = common\conditions.ini, common\materials_base.ini, common\materials_track.ini, common\conditions_seasons.ini
```

Comment:
You are not to include seasonal conditions or materials_base, these are already included with their respective files when you include conditions and track materials